docs/tempo/website/metrics-generator/service-graph-view.md

```diff
@@ -1,53 +1,60 @@
 ---
-title: Application performance management
-menuTitle: APM dashboard
+title: Service graph view
+menuTitle: Service graph view
+aliases:
+- /docs/tempo/latest/metrics-generator/app-performance-mgmt
 weight: 200
 ---
 
-# Application performance management dashboard
+# Service graph view
 
+<<<<<<<< HEAD:docs/tempo/website/metrics-generator/app-performance-mgmt.md
```

Comment:
I think this merge patch needs to be resolved

Reply:
Done. Thank you for catching that.